huaweicloud/services/dms/data_source_huaweicloud_dms_rocketmq_instances.go

```diff
@@ -50,15 +51,15 @@ instance name.`,
 			},
 			"instances": {
 				Type:        schema.TypeList,
-				Elem:        DmsRocketMQInstancesInstanceRefSchema(),
+				Elem:        RocketMQInstancesInstanceRefSchema(),
 				Computed:    true,
 				Description: `Indicates the list of DMS rocketMQ instances.`,
 			},
 		},
 	}
 }
 
-func DmsRocketMQInstancesInstanceRefSchema() *schema.Resource {
+func RocketMQInstancesInstanceRefSchema() *schema.Resource {
```

Comment:
```suggestion
func rocketMQInstancesInstanceRefSchema() *schema.Resource {
```
it seems it should be a private function

Reply:
done